content/docs/alerting/configuration.md

```diff
@@ -603,6 +604,18 @@ OpsGenie notifications are sent via the [OpsGenie API](https://docs.opsgenie.com
 [ http_config: <http_config> | default = global.http_config ]
 ```
 
+### `<responder>`
+
+```yaml
+# One of these fields should be defined.
```

Comment:
Exactly or at least?

Reply:
I think it's "exactly" but @bwplotka could confirm.

Reply:
They say nothing about this stricly so I would assume `exactly.
```
 Either id or name of each responder should be provided.You can refer below for example values.
```